src/NuGet.Core/NuGet.Packaging/ContentModel/ContentItem.cs

```diff
@@ -7,15 +7,142 @@ namespace NuGet.ContentModel
 {
     public class ContentItem
     {
-        private Dictionary<string, object> _properties;
+        // These must match ManagedCodeConventions.PropertyNames. They cannot be used since they're static and switch requires const.
+        internal const string TargetFrameworkMoniker = "tfm";
+        internal const string RuntimeIdentifier = "rid";
+        internal const string AnyValue = "any";
+        internal const string ManagedAssembly = "assembly";
+        internal const string Locale = "locale";
+        internal const string MSBuild = "msbuild";
+        internal const string SatelliteAssembly = "satelliteAssembly";
+        internal const string CodeLanguage = "codeLanguage";
+
```

Comment:
The empty space here separates the properties that match ManagedCodeConventions.PropertyNames and the ones that are more of an implementation detail rather than folder structure contract.

Reply:
```suggestion

        // These do not map to ManagedCodeConventions, they're used internally
```